### PR TITLE
Add support up to python 3.7

### PR DESCRIPTION
**Pull request by blalockma**

### CODE REVIEW

File: normalize/coll.py [hunk 1]
```diff
@@ -237,7 +241,7 @@ def coll_to_tuples(cls, coll):
                 i += 1
 
     def itertuples(self):
-        return self._values.iteritems()
+        return iter(self._values.items())
```
Comment (matthewspeck) [inline normalize/coll.py]: Should we bring this type into the 3.x world of not using iterator-based accessors?

File: normalize/property/types.py [hunk 2]
```diff
@@ -57,15 +59,15 @@ def parse_datetime(not_a_datetime):
     },
 )
 LongProperty = make_property_type(
-    "LongProperty", isa=long, trait_name="long",
+    "LongProperty", isa=six.integer_types[-1], trait_name="long",
```
Comment (matthewspeck) [inline normalize/property/types.py]: I'd really like at least a comment explaining the [dangerous] thing you're doing here.

Comment (blalockma) [reply normalize/property/types.py]: Oh yeah this garbage. Integer_types is a tuple of `(int, long)` in python2, just `(int)` in python3.
https://github.com/benjaminp/six/blob/aa4e90bcd7b7bc13a71dfaebcb2021f4caaa8432/six.py#L50

So I can always know the last one is either long or int depending on the version. Need to cut this RIGHT out the soonest we don't need py2.

**Comment by matthewspeck** [comment]
LGTM

**Pull request merged**